spotinst_sdk2/models/ocean_cd/verification_template.py

```diff
@@ -0,0 +1,422 @@
+    backoff_limit : int
+    template : TemplateSpec
+    """
+
+    def __init__(
+            self,
+            backoff_limit: int = none,
+            template: TemplateSpec = none):
+        self.backoff_limit = backoff_limit
+        self.template = template
+
+
+class Job:
+    """
+    # Arguments
+    spec : JobSpec
+    """
+
+    def __init__(
+            self,
+            spec: JobSpec = none):
+        self.spec = spec
+
+
+class Provider:
+    """
+    # Arguments
+    prometheus : Prometheus
+    new_relic : NewRelic
+    datadog : Datadog
+    cloud_watch : CloudWatch
+    web : Web
+    job : Job
+    """
+
+    def __init__(
+            self,
+            prometheus: Prometheus = none,
+            new_relic: NewRelic = none,
+            datadog: Datadog = none,
+            cloud_watch: CloudWatch = none,
+            web: Web = none,
+            job: Job = none):
+        self.prometheus = prometheus
+        self.new_relic = new_relic
+        self.datadog = datadog
+        self.cloud_watch = cloud_watch
+        self.web = web
+        self.job = job
+
+
+class VerificationMetric:
+    """
+    # Arguments
+    name : str
+    dry_run : boolean
+    interval : str
+    initial_delay : str
+    count : int
+    success_condition : str
+    failure_condition : str
+    failure_limit : int
+    consecutive_error_limit : int
+    provider : Provider
+    """
+
+    def __init__(
+            self,
+            name: str = none,
+            dry_run: bool = none,
+            interval: str = none,
+            initial_delay: str = none,
+            count: int = none,
+            success_condition: str = none,
+            failure_condition: str = none,
+            failure_limit: int = none,
+            consecutive_error_limit: int = none,
+            provider: Provider = none):
+        self.name = name
+        self.dry_run = dry_run
+        self.interval = interval
+        self.initial_delay = initial_delay
+        self.count = count
+        self.success_condition = success_condition
+        self.failure_condition = failure_condition
+        self.failure_limit = failure_limit
+        self.consecutive_error_limit = consecutive_error_limit
+        self.provider = provider
+# endregion
+
+
+# region Verification Template
+class VerificationTemplate:
+    """
+    # Arguments
+    name : str
+    args : List[VerificationArgument]
+    metrics : List[VerificationMetric]
+    created_at : str
+    updated_at : str
+    """
+
+    def __init__(
+            self,
+            name: str = none,
+            args: List[VerificationArgument] = none,
+            metrics: List[VerificationMetric] = none,
+            created_at: str = none,
+            updated_at: str = none):
+        self.name = name
+        self.args = args
+        self.metrics = metrics
+        self.created_at = created_at
+        self.updated_at = updated_at
+# endregion
+
+
+# region Client Requests
+class VerificationTemplateRequest:
```

Comment:
Rename to CreateVerificationTemplateRequest

Reply:
Done